doc/test-suite-diff.py

```diff
@@ -20,7 +21,7 @@ def start_webpage():
     head = etree.Element('head')
     html.append(head)
     title = etree.SubElement(head, 'title')
-    title.text = "Verovio test suite for Pull Request"
+    title.text = 'Verovio test suite for Pull Request'
```

Comment:
Not sure why you’re changing to single quotes?

Reply:
Mixing single with double quotes is just bad style. (PEP8)